repos/spack_repo/builtin/packages/gaudi/package.py

```diff
@@ -203,6 +220,11 @@ def cmake_args(self):
         # Gaudi@39: needs C++ >= 20, and we need to force CMake to use C++ 20 with old gcc:
         if self.spec.satisfies("@39: %gcc@:13"):
             args.append(self.define("GAUDI_CXX_STANDARD", "20"))
```

Comment:
This variable has been present since v35r0, https://gitlab.cern.ch/gaudi/Gaudi/-/commit/bfd0854d1c930d12d98e0a4b06cfbf98ba88ef82.
```suggestion
        args.append(self.define_from_variant("GAUDI_CXX_STANDARD", "cxxstd"))
```